Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -551,22 +551,33 @@ export type Props = $ReadOnly<{|
  */
 const ScrollView = createReactClass({
   displayName: 'ScrollView',
-  mixins: [ScrollResponder.Mixin],
-
+  _scrollResponder: {},
   _scrollAnimatedValue: (new AnimatedImplementation.Value(
     0,
   ): AnimatedImplementation.Value),
   _scrollAnimatedValueAttachment: (null: ?{detach: () => void}),
   _stickyHeaderRefs: (new Map(): Map<number, ScrollViewStickyHeader>),
   _headerLayoutYs: (new Map(): Map<string, number>),
+  _createScrollResponder: function() {
+    for (const key in ScrollResponder.Mixin) {
+      if (typeof ScrollResponder.Mixin[key] === 'function') {
+        this._scrollResponder[key] = ScrollResponder.Mixin[key].bind(
+          this._scrollResponder,
+        );
+      }
+    }
+    this._scrollResponder.props = this.props;
```

Comment:
This seems like it would be problematic if the props passed to ScrollView change

Reply:
Yeah, this is pretty brittle.

Reply:
One way you could get around this is to define a getter. 

```JavaScript
Object.defineProperty(this._scrollResponder, 'props', {
  get: () => {
    return this.props;
  }
})
```

But I'm not sure if that's a good idea because it'll be difficult for flow to type check this.